packages/react-native/React/Base/RCTBundleURLProvider.h

```diff
@@ -118,20 +119,38 @@ RCT_EXTERN void RCTBundleURLProviderAllowPackagerServerAccess(BOOL allowed);
  * - enableDev: Whether to keep or remove `__DEV__` blocks from the bundle.
  * - enableMinification: Enables or disables minification. Usually production bundles are minified and development
  *     bundles are not.
+ * - inlineSourceMap: Determines whether the source map is inline in the JS bundle. This is useful for debugging with Safari /JavascriptCore.
  * - modulesOnly: When true, will only send module definitions without polyfills and without the require-runtime.
  * - runModule: When true, will run the main module after defining all modules. This is used in the main bundle but not
```

Comment:
These docs in this format won't show up when you option-click with Xcode. Any issue with updating to the format Xcode prefers? That can be done my Cmd+clicking the method then selecting "Add documentation". I can imagine perhaps we don't want one method to have a different format than anything else :D

Reply:
I'll leave it alone. Would rather get the PR in sooner than worry about documentation format :)